floris/utilities.py

```diff
@@ -166,9 +166,8 @@ def wrap_180(x):
     Returns:
         np.array: Shifted values.
     """
-    x = np.where(x <= -180.0, x + 360.0, x)
-    x = np.where(x > 180.0, x - 360.0, x)
-    return x
+
+    return ( (x + 180.) % 360.) - 180.
```

Comment:
Perhaps update formatting?
`return ( (x + 180.0) % 360.0) - 180.0`

Reply:
I'd recommend Bart's suggestion of the `180.` -> `180.0` style changes plus dropping the extra space between the two leading parentheses.